charts/csi-powermax/values.yaml

```diff
@@ -53,7 +53,22 @@ version: "v2.8.0"
 
 images:
   # "driver" defines the container image, used for the driver container.
-  driverRepository: dellemc
+  driver: dellemc/csi-powermax:v2.8.0
+  csireverseproxy: dellemc/csipowermax-reverseproxy:v2.7.0
+  # CSI sidecars
+  attacher: registry.k8s.io/sig-storage/csi-attacher:v4.4.0
+  provisioner: registry.k8s.io/sig-storage/csi-provisioner:v3.6.0
+  snapshotter: registry.k8s.io/sig-storage/csi-snapshotter:v6.3.0
+  resizer: registry.k8s.io/sig-storage/csi-resizer:v1.9.0
+  registrar: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.0
+  healthmonitor: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.10.0
+  # CSM sidecars
+  replication: dellemc/dell-csi-replicator:v1.6.0
+  authorization: dellemc/csm-authorization-sidecar:v1.8.0
```

Comment:
Beyond just configuring the CSM sidecar image, other sidecar values have to be configured. I see value in consolidating the CSI sidecar images but does it make sense to consolidate the CSM sidecars? Would the additional CSM sidecar configurations get overlooked?

Reply:
The main goal is to have one place to update my private repo.

If we let the other images at the bottom it might be easy to forget ; isn't it ?